README.md

```diff
@@ -127,8 +127,9 @@ For Maven:
 
 Bringing in older versions of the supported Nexmo Client, may result in some unforseen consequences and build errors. As a result here is a list of each version targeted by each version of the starter:
 
-| Nexmo Spring Boot Starter | Nexmo Java Client |
+| Nexmo Spring Boot Starter | Nexmo/Vonage Java Client |
```

Comment:
Create a different Table for Vonage since the name is changing and there are now 2 artifacts that will exist.